bindings/nodejs/lib/types/block/output/output.ts

```diff
@@ -215,6 +224,66 @@ class AccountOutput extends ImmutableFeaturesOutput {
         this.mana = mana;
     }
 }
+
+/**
+ * Base class for state metadata outputs.
+ */
+abstract class StateMetadataOutput extends ImmutableFeaturesOutput {
```

Comment:
Why have this class? AnchorOutput are now the only outputs with a state metadata field, no?

Reply:
I just kept how it was before with the Account, but I can change it

Reply:
Would prefer that, don't see a reason to have it separated